cumulus/client/relay-chain-rpc-interface/src/lib.rs

```diff
@@ -87,12 +88,13 @@ impl RelayChainInterface for RelayChainRpcInterface {
 	async fn header(&self, block_id: BlockId) -> RelayChainResult<Option<PHeader>> {
 		let hash = match block_id {
 			BlockId::Hash(hash) => hash,
-			BlockId::Number(num) =>
+			BlockId::Number(num) => {
 				if let Some(hash) = self.rpc_client.chain_get_block_hash(Some(num)).await? {
 					hash
 				} else {
-					return Ok(None)
-				},
+					return Ok(None);
```

Comment:
nit: I'd leave it as it was, from what I see most returns are not ';' terminated.